src/openforms/js/components/admin/form_design/registrations/json/fields/ServiceSelect.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Field from 'components/admin/forms/Field';
+import FormRow from 'components/admin/forms/FormRow';
+import ReactSelect from 'components/admin/forms/ReactSelect';
+
+const ServiceSelect = ({options}) => {
+  return (
+    <FormRow>
+      <Field
+        name="service"
+        label={
+          <FormattedMessage
+            description="JSON registration options 'serviceSelect' label"
+            defaultMessage="Service"
+          />
+        }
+        helpText={
+          <FormattedMessage
+            description="JSON registration options 'serviceSelect' helpText"
+            defaultMessage="Which service to send the data to"
+          />
+        }
+      >
```

Comment:
```suggestion
      <Field
        name="service"
        label={
          <FormattedMessage
            description="JSON registration options 'serviceSelect' label"
            defaultMessage="Service"
          />
        }
        helpText={
          <FormattedMessage
            description="JSON registration options 'serviceSelect' helpText"
            defaultMessage="Which service to send the data to"
          />
        }
        required
        noManageChildProps
      >
```
* label should be bold to indicate it's a required field
* `noManageChildProps` should be the default but we can't do that yet because of all the existing code :grimacing: it avoids some of the `Field` props being manipulated and passed down to the child element (`ReactSelect` in this case)

Reply:
Also add both to `FormVariableSelect` then?